src/styles/search.css

```diff
@@ -0,0 +1,35 @@
+body {
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    height: 100vh;
+    margin: 0;
+  }
```

Comment:
You have an unchanged `index.css` from when you created the react app. You should remove the `code` css styling in there (as there are no `code` elements in your jsx) and put the changes to the `body` there, as that is where the rest of the `body` styling is